modules/ROOT/pages/_partials/mmp-deploy-to-cloudhub-2.adoc

```diff
@@ -87,6 +87,15 @@ From the command line in your project's folder, package the application and exec
 mvn clean deploy -DmuleDeploy
 ----
 
+=== Maven Snapshot Assets
+
+You can also deploy Maven snapshot assets into CloudHub 2.0. To learn more about publishing snapshot assets to Anypoint Exchange, see xref:exchange::to-publish-assets-maven.adoc#asset-lifecycle-state[Asset Lifecycle State].
```

Comment:
Same as above, let us call it Exchange snapshots (since even exchange docs does not mention 'maven snapshots'